coffee/controller/list-item-ctrl.coffee

```diff
@@ -1,7 +1,37 @@
 ListItemCtrl = ($scope, githubService) ->
 
   $scope.getNotifications = () ->
-    githubService.notifications($scope.repo)
+    answer = githubService.notifications($scope.repo)
+    if !answer.value
```

Comment:
will need to handle this as a promise.
if you check `githubService` and `apiService`, this function will actually return a promise.